Compilation of lib/QoreEllipsesNode.cpp breaks in non-scu builds

Compilation of lib/QoreEllipsesNode.cpp breaks in non-scu builds with the message:
``lib/QoreEllipsesNode.cpp: In member function 'virtual int QoreEllipsesNode::parseInitImpl(QoreValue&, QoreParseContext&)':``
``lib/QoreEllipsesNode.cpp:42:27: error: incomplete type 'qore_program_private' used in nested name specifier
     qore_program_private::get(*getProgram())->makeParseException(*loc, "ILLEGAL-ELLIPSES",``

Solution: include qore/intern/qore_program_private.h in QoreEllipsesNode.cpp

--- a/lib/QoreEllipsesNode.cpp
+++ b/lib/QoreEllipsesNode.cpp
@@ -29,6 +29,7 @@
 */
 
 #include <qore/Qore.h>
+#include "qore/intern/qore_program_private.h"
 
 QoreString QoreEllipsesNode::staticEllipses("...");
 

